nimterop/ast.nim

```diff
@@ -134,6 +134,9 @@ proc printNimHeader*() =
 #   $2 $3
 
 {.experimental: "codeReordering".}
+{.hint[ConvFromXtoItselfNotNeeded]: off.}
```

Comment:
this is hiding issue under a rug though... we should at least keep an issue open to track this or at least a TODO in code